dynamic_programming/golmine.py

```diff
@@ -0,0 +1,32 @@
+# Goldmine Problem.
+
+#Question Problem : https://leetcode.com/problems/path-with-maximum-gold/
+
+n = int(input().strip())
+m = int(input().strip())
+arr = [[0]*m for _ in range(n)]
+print(arr)
```

Comment:
Algorithms should not have any side-effects. Consider creating two functions: one for the algorithm itself, and another that invokes this function and prints output.